src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -209,17 +212,21 @@ function enableQuestion(number) {
 function enableNewQuestion() {
     var newQnSuffix = "New";
     var number = "-1";
-    $('#questionTable'+newQnSuffix).find('text,button,textarea,select,input').
+    
+    var $currentQuetsionTablSuffix = $('#questionTable' + newQnSuffix);
+    var $currentQuestionTableNumber = $('#questionTable' + number);
+    
+    $currentQuetsionTablSuffix.find('text,button,textarea,select,input').
         not('[name="receiverFollowerCheckbox"]').
         not('.disabled_radio').
         removeAttr("disabled", "disabled");
-    $('#questionTable'+newQnSuffix).find('.removeOptionLink').show();
-    $('#questionTable'+newQnSuffix).find('.addOptionLink').show();
+    $currentQuetsionTablSuffix.find('.removeOptionLink').show();
+    $currentQuetsionTablSuffix.find('.addOptionLink').show();
```

Comment:
Nope, it's to differentiate between a jQuery wrapped variable and a variable that is not wrapped, in our code base there are many elements that have already been wrapped and gets re-wrapped again and again, it can affect performance and is also confusing.